include/revng/Pipes/Ranks.h

```diff
@@ -87,4 +87,10 @@ inline auto
 inline auto
   ArtificialStruct = defineRank<"artificial-struct",
                                 model::RawFunctionDefinition::Key>(Binary);
+
+/// Rank for stack arguments of raw functions
+inline auto
+  RawStackArguments = defineRank<"raw-stack-arguments",
+                                 model::RawFunctionDefinition::Key>(Binary);
```

Comment:
What is this for?

Reply:
For the RFT's `void function(..., Type stack_arguments)` thing. So that it's ctrl+clickable.